bundle/Service/ScheduledVisibilityService.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Service;
+
+use DateTime;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+use Ibexa\Contracts\Core\Repository\Values\Content\Field;
+use Ibexa\Core\FieldType\Date\Value as DateValue;
+use Ibexa\Core\FieldType\DateAndTime\Value as DateAndTimeValue;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\Configuration\ScheduledVisibilityConfiguration;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\Enums\VisibilityUpdateResult;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\Exception\InvalidStateException;
+use Netgen\Bundle\IbexaScheduledVisibilityBundle\ScheduledVisibility\Registry;
+use OutOfBoundsException;
+
+final class ScheduledVisibilityService
+{
+    public function __construct(
+        private readonly ScheduledVisibilityConfiguration $configurationService,
+        private readonly Registry $registry,
+    ) {}
+
+    /**
+     * @throws OutOfBoundsException
```

Comment:
`OutOfBoundsException` is not thrown in this method.

Reply:
fixed infa153b003652a49ea510b3b043c68e87d73d325f